vms/platformvm/warp/validator.go

```diff
@@ -154,3 +159,17 @@ func AggregatePublicKeys(vdrs []*Validator) (*bls.PublicKey, error) {
 	}
 	return bls.AggregatePublicKeys(pks)
 }
+
+// GetCanonicalValidatorSetFromState returns the canonical validator set given a validators.State, pChain height and a sourceChainID.
+func GetCanonicalValidatorSetFromState(ctx context.Context,
```

Comment:
`GetCanonicalValidatorSet` vs `GetCanonicalValidatorSetFromState`. Based on the names, I would expect `GetCanonicalValidatorSet` not to take in the `State` and for `GetCanonicalValidatorSetFromState` to take in the `State`.

They both take in states though... One of them taking in a chainID + state which can convert the chainID into a subnetID. The other taking in a subnetID + state.

Could we improve the names of these functions?

Reply:
ok, done